ls1intum/Artemis
# [Exam] Instructor code editor

### Checklist

### Description

- [x] Decouple row buttons displayed in exercise group view from text exercise row buttons of course->exercises
- [x] Enable isAtLeastInstructor and  isAtLeastTutor checks in the exerciseGroup view. Use isAtLeastInstructor for every navigation element there for now
- [x] Enable instructor code editor for exam programming exercises. Disabled assignment repository actions in exam mode for now.

### Steps for Testing


_Check that all course text exercise buttons are still there and work_
1. Navigate to Course Management->Exercises
--> Row buttons for text exercises are displayed correctly and work

_Check that all row buttons in the exercise group view are still there and work_
1. Navigate to Course Management->Exams->Exercise Groups
--> Row buttons for all exercise types are displayed correctly and work

_Check the instructor code editor_
1. Navigate to Course Management->Exams->Exercise Groups. Open the Instructor code editor of a programming exercise by clicking on the button 'Edit in editor'
2. Switch between the different participations and play around here
--> Everything should work as in the course programming exercise instructor code editor

### Screenshots


![Capture](https://user-images.githubusercontent.com/16407766/86948811-009b2980-c14e-11ea-80b1-fa6f2d636678.PNG)
